archive_explorer.md

```diff
@@ -182,6 +182,57 @@
         await updateMap();
         loading.value = false;
       };
+      
+      // computed properties to calculate package statistics
+      const packageStats = computed(() => {
+        if (!selectedPackage.value) return null;
+
+        let numMale = 0;
+        let numFemale = 0;
+        let totalSNPs = 0;
+        let numModern = 0;
+        let numAncient = 0;
+
+        for (const sample of getSamplesForPackage(selectedPackage.value.packageTitle)) {
+          const sex = sample.additionalJannoColumns.find((col) => col[0] === 'Genetic_Sex');
+          const snps = sample.additionalJannoColumns.find((col) => col[0] === 'Nr_SNPs');
+          const dateType = sample.additionalJannoColumns.find((col) => col[0] === 'Date_Type');
+          
+
+          if (sex && snps && dateType) {
+            const sexValue = sex[1].toLowerCase();
+            const snpsValue = parseFloat(snps[1]);
+            const dateTypeValue = dateType[1].toLowerCase();
+
+            if (!isNaN(snpsValue)) {
+              totalSNPs += snpsValue;
+            }
+
+            if (sexValue === 'm') {
+              numMale++;
+            } else if (sexValue === 'f') {
+              numFemale++;
+            }
+
+            if (dateTypeValue === 'modern') {
+              numModern++;
+            } else {
+              numAncient++;
+            }
```

Comment:
What happens if the Date_Type value is `NA`? Unfortunately we have a number of these in the PCA.